docs/developer-docs/latest/update-migration-guides/migration-guides.md

```diff
@@ -26,11 +27,19 @@ If you were to upgrade your version from `3.2.3` to `3.6.1`, you would have to f
 4. Migration guide from 3.4.x to 3.4.4.
 5. [Update Strapi guide.](update-version.md)
 
-## v4-Stable guides
+## v4 stable guides
+
+::: callout 🚧 Upcoming migration guides
+This section is still a work in progress. Each of the 3 planned migration guides (i.e. plugins, code and data migration guides) will be released separately. In the meantime, feel free to ask for help on the [forum](https://forum.strapi.io/) or on the community [Discord](https://discord.strapi.io).
```

Comment:
missing comma after code, should be `(i.e. plugins, code, and data migration guides)`